src/components/DashboardLayout/SettingsLayout.tsx

```diff
@@ -49,17 +52,19 @@ export const SettingsLayout = () => {
         setTabIndex,
       }}
     >
-      <div className="flex grow flex-row justify-center gap-[1vw]">
-        <div className="flex h-full grow basis-60 flex-row flex-nowrap items-stretch justify-end">
-          <div className="h-full w-[16rem] max-w-[16rem] child:h-full">
-            <SettingsLeftNav />
-          </div>
-        </div>
-        <div className="h-full min-w-0 grow basis-[1024px] child:h-full">
-          <div className="max-w-[1024px]">
-            <Outlet />
-          </div>
-        </div>
+      <ResizePanel
+        direction="e"
+        maxSize={500}
+        minSize={200}
+        size={leftNavSize}
+        validateSize={false}
+        onResize={setLeftNavSize}
+        containerClassNames="min-h-0"
+      >
+        <SettingsLeftNav className="bg-gray-850" />
```

Comment:
Sync behavior between setting left nav and normal left nav.
They should be both resizable, or not
Also currently the setting left nav is missing its right border